index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+<body>
+
+<section class="header">
+    
+    <div class="Top-Nav">
+        <div class="logo">
+          My Logo
+        </div>
+        <div class="Item-Nav">
+            
+            <a href="">Portfolio</a>
+            <a href="">About</a>
+            <a href="">Contact</a>
+        </div>
+
+    </div>
+    <div class="content">
+        <h1>I'm Fithamlak <br/> glad to see you</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p>LET’S CONNECT</p>
+        <div class="social-media-icons">
```

Comment:
- [ ] According to the project requirements, you are required to use `ul` and `li` to group social icons.